<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hw5</class>
 <widget class="QDialog" name="hw5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>10</y>
     <width>481</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>390</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CMY">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CMY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_HSI">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>HSI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_XYZ">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>XYZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LAB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>L*A*b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_YUV">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>YUV</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_colorbar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>256</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pseudo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pseudo color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pseudocolor">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_RGB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
